local_storage_utils.ts

```diff
@@ -0,0 +1,7 @@
+import { Recipe } from './types'
+
+export const setToLocalStorage = (key:string, data:Array<string>|Array<Recipe>) => {
```

Comment:
can be `any` as indeed storage can save any data. `any` is forbidden unless you know what you're doing